timemachine/cpp/src/kernels/k_flat_bottom_bond.cuh

```diff
@@ -110,11 +110,8 @@ void __global__ k_flat_bottom_bond(
     if (u) {
         RealType u_real = compute_flat_bottom_energy<RealType>(k, r, rmin, rmax);
 
-        // cast float -> fixed
-        auto sum_u = FLOAT_TO_FIXED_BONDED<RealType>(u_real);
-
-        // atomic add to u array
-        atomicAdd(u + src_idx, sum_u);
+        // Set idx to force
```

Comment:
nit: comment not clear to me. What is "force" here?

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1090/commits/f9e29f743413a8d51736473e77c0fc2395d585d9